Orange/data/domain.py

```diff
@@ -58,69 +69,101 @@ def __init__(self, source, destination):
 
 
 class Domain:
-    def __init__(self, attributes, class_vars=None, metas=None, source=None):
+    def __init__(self, attributes=[], class_vars=[], metas=[], source=None):
```

Comment:
[This](https://github.com/biolab/orange3/pull/978/files#diff-698efb0bca09486f1b8892fac56a8661R72) better? :smiley: